puppet-checks/src/main/java/com/iadams/sonarqube/puppet/checks/FileModeCheck.java

```diff
@@ -60,22 +63,36 @@ public void init() {
 
   @Override
   public void visitNode(AstNode node) {
-    if(node.getTokenValue().equals("file")) {
+    if ("file".equals(node.getTokenValue())) {
       for (AstNode body : node.getDescendants(PuppetGrammar.RESOURCE_INST)) {
```

Comment:
I'd need to double check if the RESOURCE_INST are first level children but probably, i'll check later.